nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/TestTreeIndexing.java

```diff
@@ -96,6 +99,7 @@ public void assertNumberOfCommandProcessed(int processed) throws Exception {
      * Wait for async worker completion then wait for indexing completion
      */
     public void waitForCompletion() throws Exception {
+        bulk.await(Duration.ofSeconds(20));
         workManager.awaitCompletion(20, TimeUnit.SECONDS);
```

Comment:
Can you replace the two calls above by `TransactionalFeature#nextTransaction(Duration)` ? or commit of transaction is problematic ?